src/main/java/org/tarantool/MsgPackLite.java

```diff
@@ -72,13 +72,6 @@
 
     public void pack(Object item, OutputStream os) throws IOException {
         DataOutputStream out = new DataOutputStream(os);
-        if (item instanceof Callable) {
```

Comment:
U could see that enum Key use Callable interface to provide id. Did u test code after PR?